molgenis-platform-integration-tests/src/test/java/org/molgenis/integrationtest/utils/BootstrapTestUtils.java

```diff
@@ -0,0 +1,111 @@
+package org.molgenis.integrationtest.utils;
+
+import org.molgenis.data.EntityFactoryRegistrar;
+import org.molgenis.data.RepositoryCollectionBootstrapper;
+import org.molgenis.data.SystemRepositoryDecoratorFactoryRegistrar;
+import org.molgenis.data.jobs.JobFactoryRegistrar;
+import org.molgenis.data.meta.system.SystemEntityTypeRegistrar;
+import org.molgenis.data.meta.system.SystemPackageRegistrar;
+import org.molgenis.data.platform.bootstrap.SystemEntityTypeBootstrapper;
+import org.molgenis.data.postgresql.identifier.EntityTypeRegistryPopulator;
+import org.molgenis.data.settings.SettingsPopulator;
+import org.molgenis.data.transaction.TransactionManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.support.TransactionTemplate;
+import org.springframework.web.context.WebApplicationContext;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.molgenis.data.postgresql.PostgreSqlRepositoryCollection.POSTGRESQL;
+import static org.molgenis.security.core.runas.RunAsSystemAspect.runAsSystem;
+
+@Component
+public class BootstrapTestUtils
+{
+	private static final Logger LOG = LoggerFactory.getLogger(BootstrapTestUtils.class);
+
+	@Autowired
+	private RepositoryCollectionBootstrapper repoCollectionBootstrapper;
+	@Autowired
+	private SystemEntityTypeRegistrar systemEntityTypeRegistrar;
+	@Autowired
+	private SystemPackageRegistrar systemPackageRegistrar;
+	@Autowired
+	private EntityFactoryRegistrar entityFactoryRegistrar;
+	@Autowired
+	private SystemEntityTypeBootstrapper systemEntityTypeBootstrapper;
+	@Autowired
+	private SystemRepositoryDecoratorFactoryRegistrar systemRepositoryDecoratorFactoryRegistrar;
+	@Autowired
+	private JobFactoryRegistrar jobFactoryRegistrar;
+	@Autowired
+	private TransactionManager transactionManager;
+	@Autowired
+	private SettingsPopulator settingsPopulator;
+
+	public void bootstrap(WebApplicationContext context)
+	{
+		ContextRefreshedEvent event = mock(ContextRefreshedEvent.class);
+		when(event.getApplicationContext()).thenReturn(context);
+
+		TransactionTemplate template = new TransactionTemplate();
+		template.setTransactionManager(transactionManager);
+		try
+		{
+			runAsSystem(() -> initialize(template, event));
+		}
+		catch (Exception unexpected)
+		{
+			LOG.error("Error bootstrapping tests!", unexpected);
+			throw new RuntimeException(unexpected);
+		}
+	}
+
+	private void initialize(TransactionTemplate template, ContextRefreshedEvent event)
+	{
+		template.execute((action) ->
+		{
+			LOG.info("Bootstrapping registries ...");
```

Comment:
Add a TODO with a reference to https://github.com/molgenis/molgenis/issues/5934 and update the class name in the issue to the new class name

Reply:
This is done, it is added above the method.